readers/influxdb/messages.go

```diff
@@ -35,14 +39,21 @@ func New(client influxdata.Client, database string) readers.MessageRepository {
 }
 
 func (repo *influxRepository) ReadAll(chanID string, offset, limit uint64, query map[string]string) (readers.MessagesPage, error) {
+	measurement, ok := query[format]
+	if !ok {
+		measurement = defMeasurement
+	}
+	// Remove format filter and format the rest properly.
+	delete(query, format)
 	condition := fmtCondition(chanID, query)
-	cmd := fmt.Sprintf(`SELECT * FROM messages WHERE %s ORDER BY time DESC LIMIT %d OFFSET %d`, condition, limit, offset)
+
+	cmd := fmt.Sprintf(`SELECT * FROM %s WHERE %s ORDER BY time DESC LIMIT %d OFFSET %d`, measurement, condition, limit, offset)
 	q := influxdata.Query{
 		Command:  cmd,
 		Database: repo.database,
 	}
 
-	ret := []senml.Message{}
+	ret := []interface{}{}
```

Comment:
Can we create custom type on domain level, like in `transformers` and use it like `[]MsgPaylod` ? With probably better naming :)